crates/common/src/job.rs

```diff
@@ -40,26 +41,28 @@ impl Job {
             .verify(&message_hash, &Signature { r: self.signature_r, s: self.signature_s })
             .unwrap()
     }
+
+    pub fn reward(&self) -> U256 {
+        self.job_data.reward
+    }
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct JobData {
-    pub reward: u64,
+    pub reward: U256,
```

Comment:
I had switched the type into U256 as it's compatible with `deposit` function call amount input. Wondering if we could also use U256 in bootloader input type @Okm165

Reply:
@rkdud007 I agree.
I also confirmed that on other starknet smart-contracts values/amounts are `u256` so it stays this way for sure.
Feel free to change it and other necessary things on a rust side.
I will handle the bootloader compatibility in this PR.